src/mail-app/mail/view/MailViewer.ts

```diff
@@ -112,6 +112,7 @@ export class MailViewer implements Component<MailViewerAttrs> {
 	}
 
 	onremove({ attrs }: Vnode<MailViewerAttrs>) {
+		this.loadAllListener.end(true)
```

Comment:
`onremove` is called any time the view is removed from DOM, right? So if you switch to another view and then come back the listener would stay detached. We need some better strategy.

Why is `loadCompleteNotification` even a stream? Should it be a Promise or something? Although it will have issues with unsubscribing I guess, unless we race or something…